build.sbt

```diff
@@ -404,7 +404,12 @@ lazy val parquet: Project = project
     description := "Magnolia add-on for Apache Parquet",
     libraryDependencies ++= Seq(
       "org.apache.hadoop" % "hadoop-client" % hadoopVersion % Provided,
-      "org.apache.parquet" % "parquet-avro" % parquetVersion % Provided,
+      "org.apache.parquet" % "parquet-avro" % parquetVersion % Provided excludeAll (
+        // parquet-avro depends on avro 1.10.x
+        ExclusionRule("org.apache.avro", "avro"),
+        ExclusionRule("org.apache.avro", "avro-compiler")
```

Comment:
prefer `dependencyOverrides` in this case since we do not exclude but force version

Reply:
Done